core/src/main/kotlin/behavior/ChatInputCommandBehavior.kt

```diff
@@ -27,9 +30,9 @@ public interface GuildChatInputCommandBehavior : ChatInputCommandBehavior, Guild
 
 
 public interface GlobalChatInputCommandBehavior : ChatInputCommandBehavior,GlobalApplicationCommandBehavior {
-
+     @OptIn(KordUnsafe::class)
      override suspend fun edit(builder: suspend ChatInputModifyBuilder.() -> Unit): GlobalChatInputCommand {
```

Comment:
Can we find a way to use `GlobalChatInputModifyBuilder` here as the extension receiver (also for user/message)? Overriding breaks this.

Reply:
a not so elegant way would be to provide additional overloads as inline extensions

Reply:
or we just give up the base interface methods like in `MultiApplicationCommandBuilder`

Reply:
> a not so elegant way would be to provide additional overloads as inline extensions (might not even work as members take precedence over extensions)

It does work, but the ide will always rank the non extension one first, so no one will find it

> or we just give up the base interface methods like in MultiApplicationCommandBuilder

I don't think that's a good idea

Reply:
the solution might be to have members in the subtypes and an extension for the base type which takes the more generic builder